scripts/test_mail-alarm.py

```diff
@@ -8,6 +8,10 @@
 import sys
 import unittest
 import mock
+import pytest
+
+if sys.version_info > (2, ):
```

Comment:
This will be easy to miss, ideally there's a task that tracks all the lines that need to be deleted before merging the branch to master. This could be done in ./qualitygate.sh

Reply:
Sure. Track this in https://issues.citrite.net/browse/CA-392725